docs/source/data_recipes/CDS_toolbox_template.md

```diff
@@ -26,76 +26,107 @@ At present, the pre-processing does not include scaling or topographic adjustmen
 
 NOTE: You need to create a user account to access all data and functionalities.
 
+The data handling for simulations is managed by the {mod}`~virtual_rainforest.core.data`
+module and the {class}`~virtual_rainforest.core.data.Data` class, which provides the
+data loading and storage functions for the Virtual Rainforest. The data system is
+extendable to provide support for different file formats and axis validation but that is
+beyond the scope of this document.
+
 ## Climate input variables
 
-The abiotic module of the virtual rainforest requires the following climate input
+The simple abiotic module of the Virtual Rainforest requires the following climate input
 variables (or derivatives) at each time step (default: monthly means):
 
 * Air temperature (typically 2m; mean, minimum, and maximum)
-* Air humidity (typically 2m; relative or specific humidity)
-* Air pressure (typically mean sealevel or surface pressure)
-* Wind speed (typically 10m)
+* Relative humidity (typically 2m; relative or specific humidity)
+* Atmospheric pressure (typically mean sealevel or surface pressure)
 * Precipitation
-* Top of atmosphere short-wave downward radiation
-* CO2 concentration (for future projections)
-* optional: soil temperature and soil moisture for initialisation
+* optional: atmospheric $\ce{CO_{2}}$ concentration, soil temperature, soil moisture
```

Comment:
I would maybe split this into a separate paragraph bullet point list, as find it a bit a weird to have optional inputs listed in a bullet point list of the required variables